code/game/objects/items/storage/uplink_kits.dm

```diff
@@ -235,6 +237,13 @@
 			
 		if(KIT_TRAITOR_2006) //A kit so old, it's probably older than you. //This bundle is filled with the entire unlink contents traitors had access to in 2006, from OpenSS13. Notably the esword was not a choice but existed in code.
 			/obj/item/storage/toolbox/emergency/old/ancientbundle //Items fit neatly into a classic toolbox just to remind you what the theme is.
+			
+		if(KIT_RED_IS_SUS)
+			new /obj/item/clothing/head/helmet/space/syndicate(src)
+			new /obj/item/clothing/suit/space/syndicate(src)
+			new /obj/item/dnainjector/ventcrawler(src)
+			new /obj/item/kitchen/knife/combat(src)
+			new /obj/item/gun/ballistic/automatic/pistol/m1911(src)
```

Comment:
a spare mag for this for much easier tactical reloading would be nice.

Reply:
Ventcrawl is already honestly pretty powerful since you can more or less ambush people with little warning. I think the weapons are strong enough. You can reload the M1911 through autolathe bullets if need be.

Reply:
I might give it thermals so they can see people they get the jump on though

Reply:
Actually nah, it's fine as is. Nothing is stopping you from ventcrawling right into the armoury and giving the Warden a hard time.